docs/user-guide/security-features.md

```diff
@@ -1,156 +1,228 @@
 
 However, you have the flexibility to make explicit changes to the policy if needed. This empowers you to customize the policy to align with specific requirements or preferences.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/environment-security-policy.jpg)
+![Figure 11: Configuring Environment Security Policy](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/new/environment-security-policy-1.jpg)
+
+---
 
 ## Configure Application Security Policy
 
-The `Application Security Policy` operates on a similar principle as other policies. However, in the `Application Security Policy`, the policy is determined by both the Environment option and the Application option.
+The Application Security Policy operates on a similar principle as other policies and offers four options:
+
+* Block always
+* Block if fix is available
+* Allow
+* Inherit
+
+However, in the Application Security Policy, the policy is determined by both the Environment option and the Application option.
 
 When modifying the policy within a development environment, the changes will be applied to all applications within that specific development environment. This means that any adjustments made to the policy settings will be consistently applied across all applications associated with that particular development environment.
 
 This approach ensures uniformity and streamlined management of security policies within specific environments and their corresponding applications.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/application-security-policy.jpg)
+![Figure 12: Configuring Application Security Policy](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/new/application-security-policy-1.jpg)
 
-## Block or Allow Specific CVE Policies.
+---
 
-To block or allow specific Common Vulnerabilities and Exposures (CVE) policies, simply click `Add CVE Policy`.
+## Block or Allow Specific CVE Policies
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/add-cve-policy.jpg)
+To block or allow specific Common Vulnerabilities and Exposures (CVE) policies, simply click **Add CVE Policy**.
+
+![Figure 13: Adding CVE Policy](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/new/add-cve-policy-1.jpg)
 
 A window will appear where you can enter the CVE ID and select whether to allow or block it.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/cve-popup.jpg)
+![Figure 14: Allowing/Blocking a CVE ID](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/new/cve-popup.jpg)
```

Comment:
'Block if fix is available' option is missing in the Add CVE Modal. Refer screenshot:
<img width="680" alt="Screenshot 2024-01-14 at 2 18 01 PM" src="https://github.com/devtron-labs/devtron/assets/73632042/d8fb2f8e-f139-4985-afe3-382f0fc80da6">